test/test_utils.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+import pandas as pd
+
+from carla import utils
+
+
+def test_succes_rate():
+    df = pd.DataFrame([1, 1, 1, np.nan, np.nan])
+    actual_rate, actual_indices = utils.success_rate_and_indices(df)
+    expected_rate = 0.6
+    expected_indices = [0, 1, 2]
+
+    assert actual_rate == expected_rate
+    assert (actual_indices == expected_indices).all()
```

Comment:
Some tests for the remaining measurements would be nice.
Redundancy and time should be pretty hard, @indyfree is there a common way to test such functions, where we can't define an expected value?

Reply:
For redundancy the value should be deterministic if the model and data is fixed right? For time i'd skip testing specific values.

In general pytest has the approx function, where we expect slight deviation of values: https://docs.pytest.org/en/6.2.x/reference.html#pytest-approx

Reply:
Refactored test into `test_success_rate.py`